services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1247,6 +1247,9 @@ module.exports = {
         { 'relatedTo.$id': companyId },
       ];
 
+      const includeSponsored = getAsArray(query, 'type.$in').includes('Webinar');
+      if (includeSponsored) query.$or.push({ sponsors: companyId });
```

Comment:
Nevermind on the index, but lets just change this to include `sponsors` in the `$or` query above -- no need to add this separately.

Reply:
Nevermind on the nevermind. Also need index.